Source/Mosa.Platform.Internal.x86/Runtime.cs

```diff
@@ -330,6 +329,31 @@ public static void Fault(uint code)
 			return null;
 		}
 
+		public static uint GetMethodStartAddress(uint address)
```

Comment:
This is redundant as you can achieve the same thing using:
var methodStartAddress = (uint)(GetMethodDefinition(address)->Method);